[KV]Update get-started.md (#13133)
cloudflare/cloudflare-docs: content/kv/get-started.md

4. Enter a name for your namespace. 
5. Select **Add**.

> {{<Aside type="note">}}KV namespaces prior to version 7 cannot be edited via the dashboard. To do so use the [KV API](/kv/api/){{</Aside>}}

Reviewer: ```suggestion
{{<Aside type="note">}}
KV namespaces prior to version 7 cannot be edited via the Cloudflare dashboard. To edit KV namespaces, use the [KV API](/kv/api/)
{{</Aside>}}
```
